app/services/search/tag.rb

```diff
@@ -0,0 +1,84 @@
+module Search
+  class Tag
+    INDEX_NAME = "tag_#{Rails.env}".freeze
+    INDEX_ALIAS = "tag_#{Rails.env}_alias".freeze
+
+    class << self
+      def index(tag_id, serialized_data)
+        SearchClient.index(
+          id: tag_id,
+          index: INDEX_ALIAS,
+          body: serialized_data,
+        )
+      end
+
+      def get(tag_id)
+        SearchClient.get(id: tag_id, index: INDEX_ALIAS)
+      end
+
+      def create_index(index_name = INDEX_NAME)
```

Comment:
can we use keyword arguments for everything? I'm already envisioning a situation where in `add_alias` we swap the parameters by chance :D

Reply:
Never been a huge fan of keyword arguments but you make a good point

Reply:
I love them! They are so helpful and self documenting. BTW Ruby 2.7 is definitely leaning towards keyword arguments for the future Ruby 3: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/